charts/remark42/templates/secureconfig.yaml

```diff
@@ -12,9 +12,11 @@ data:
 {{- end }}
 {{- with .Values.settings.smtp }}
   {{- if .enabled }}
+  {{- if .name }}
```

Comment:
This will check for a valid name, but what about missing password?
I would prefer to have the global .enabled switch and if this is "on" then a valid name and/or password needs to be provided.
Alternative would be to check with `required` syntax of helm to make sure name and/or password are valid.